examples/notebook/notebook.rs

```diff
@@ -0,0 +1,44 @@
+use gtk::glib;
+use gtk::prelude::*;
+use gtk::{IconSize, Orientation, ReliefStyle, Widget};
+
+pub struct Notebook {
+    pub notebook: gtk::Notebook,
```

Comment:
This should really be a subclass or not? :)

Reply:
Very good point, will change that.

Reply:
`gtk::Notebook` is not subclassable in GTK3, if I interpret this error message correctly

```
the trait bound `gtk::Notebook: IsSubclassable<imp::Notebook>` is not satisfied
the trait `IsSubclassable<imp::Notebook>` is not implemented for `gtk::Notebook`
```

Reply:
Indeed